pictagram/spec/feature/post_feature_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+require 'web_helper'
+feature 'posts' do
+  context 'no posts have been added' do
+    scenario 'should display a prompt to add a post' do
+      visit '/posts'
+      expect(page).to have_content 'No posts yet'
+      expect(page).to have_link 'Add a post'
+    end
+  end
+
+  context 'posts have been added' do
+    before do
+      Post.create(comment: 'My first post')
+    end
+
+    scenario 'display posts' do
+      visit '/posts'
+      expect(page).to have_content('My first post')
+      expect(page).not_to have_content('No posts yet')
+    end
+  end
+
+  context 'creating posts' do
+    scenario 'prompts user to fill out a form, then displays the new post' do
+      visit '/posts'
+      sign_up
+      click_link 'Add a post'
+      fill_in 'post[comment]', with: 'My Fave Breakfast!'
+      click_button 'Create Post'
+      expect(page).to have_content 'My Fave Breakfast!'
+      expect(current_path).to eq '/posts'
+    end
+
+
+    context 'an invalid post' do
+    it 'does not let you submit a post comment that is too short' do
+      visit '/posts'
+      sign_up
+      click_link 'Add a post'
+      fill_in 'post[comment]', with: 'M'
+      click_button 'Create Post'
+      expect(page).not_to have_css 'h2', text: 'M'
+      expect(page).to have_content 'error'
+    end
+  end
```

Comment:
Lint/BlockAlignment: `end` at 46, 2 is not aligned with `context 'an invalid post' do` at 36, 4.